client/extensions/woocommerce/app/products/product-update.js

```diff
@@ -0,0 +1,166 @@
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { editProduct, editProductAttribute, createProductActionList } from 'woocommerce/state/ui/products/actions';
+import { editProductCategory } from 'woocommerce/state/ui/product-categories/actions';
+import { getActionList } from 'woocommerce/state/action-list/selectors';
+import { getCurrentlyEditingProduct } from 'woocommerce/state/ui/products/selectors';
+import { getProductVariationsWithLocalEdits } from 'woocommerce/state/ui/products/variations/selectors';
+import { editProductVariation } from 'woocommerce/state/ui/products/variations/actions';
+import { fetchProduct } from 'woocommerce/state/sites/products/actions';
+import { fetchProductCategories } from 'woocommerce/state/sites/product-categories/actions';
+import { getProductCategoriesWithLocalEdits } from 'woocommerce/state/ui/product-categories/selectors';
+import { createProduct } from 'woocommerce/state/sites/products/actions';
+import ProductForm from './product-form';
+import ProductHeader from './product-header';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+class ProductUpdate extends React.Component {
+	static propTypes = {
+		params: PropTypes.object,
+		className: PropTypes.string,
+		product: PropTypes.shape( {
+			id: PropTypes.isRequired,
+		} ),
+		fetchProduct: PropTypes.func.isRequired,
+		fetchProductCategories: PropTypes.func.isRequired,
+		editProduct: PropTypes.func.isRequired,
+		editProductCategory: PropTypes.func.isRequired,
+		editProductAttribute: PropTypes.func.isRequired,
+		editProductVariation: PropTypes.func.isRequired,
+	};
+
+	componentDidMount() {
+		const { params, product, site } = this.props;
+		const productId = Number( params.product );
+
+		if ( site && site.ID ) {
+			if ( ! product ) {
+				this.props.fetchProduct( site.ID, productId );
+				this.props.editProduct( site.ID, { id: productId }, { type: 'simple' } );
+			}
+			this.props.fetchProductCategories( site.ID );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		const { params, site } = this.props;
+		const productId = Number( params.product );
+		const newSiteId = newProps.site && newProps.site.ID || null;
+		const oldSiteId = site && site.ID || null;
+		if ( oldSiteId !== newSiteId ) {
+			this.props.fetchProduct( newSiteId, productId );
+			this.props.editProduct( newSiteId, { id: productId }, { type: 'simple', } );
+			this.props.fetchProductCategories( newSiteId );
+		}
+	}
+
+	componentWillUnmount() {
+		// TODO: Remove the product we added here from the edit state.
+	}
+
+	onTrash = () => {
+		// TODO: Add action dispatch to trash this product.
+	}
+
+	onSave = () => {
+		const { product, translate } = this.props;
+
+		const successAction = successNotice(
+			translate( '%(product)s successfully updated.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **57** times:
`translate( '%1$s updated successfully.' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woo-product-update-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).